dashboard/tests/test_helpers.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from dashboard.helpers import shorten_node_key
+
+class TestHelpers(unittest.TestCase):
+    def test_shorten_node_key_success(self):
+        self.assertEqual(
+            '0x1234..abcd',
```

Comment:
Would be goo to have format similar to Git hashes, so that I can search by begening/ending of node identity